src/pages/LandingPage.tsx

```diff
@@ -15,16 +15,18 @@ export const LandingPage: React.FC = () => {
         <Row>
           <Col>
             <img className={styles.LandingPageFullWidthImagesContainer} src={landingPage1} />
-            <p className={styles.LandingPageOverlayText}>
+            <div className={styles.LandingPageOverlayText}>
+              <p className={styles.OverlayTextMargin}>
               <div className={styles.UnderlineBlockWrapper}>
                 <strong>Millions</strong>
                 <div className={styles.UnderlineBlock}></div>
               </div>{' '}
-              of deaths from COVID-19 are underreported.
+              of deaths from COVID-19 are underreported.</p>
```

Comment:
need to be linted properly

if you're using vscode as your editor, try checking the settings I documented in README if you want the editor to autolint at every save: [link](https://github.com/UncountedDeaths/UncountedDeaths.github.io/tree/react_transfer#vscode-configurations-eslint--prettier)

Reply:
I've added it to my settings.json